taskana-adapter-camunda-listener/src/test/java/pro/taskana/adapter/camunda/dto/ReferencedTaskTest.java

```diff
@@ -0,0 +1,206 @@
+    ReferencedTask referencedTask = new ReferencedTask();
+    referencedTask.setDescription(theValue);
+    assertThat(theValue).isEqualTo(referencedTask.getDescription());
+  }
+
+  @Test
+  void should_returnOwner_when_OwnerWasSet() {
+    ReferencedTask referencedTask = new ReferencedTask();
+    referencedTask.setOwner(theValue);
+    assertThat(theValue).isEqualTo(referencedTask.getOwner());
+  }
+
+  @Test
+  void should_returnPriority_when_PriorityWasSet() {
+    ReferencedTask referencedTask = new ReferencedTask();
+    referencedTask.setPriority(theValue);
+    assertThat(theValue).isEqualTo(referencedTask.getPriority());
+  }
+
+  @Test
+  void should_returnSuspended_when_SuspendedWasSet() {
+    ReferencedTask referencedTask = new ReferencedTask();
+    referencedTask.setSuspended(theValue);
+    assertThat(theValue).isEqualTo(referencedTask.getSuspended());
+  }
+
+  @Test
+  void should_returnSystemUrl_when_SystemUrlWasSet() {
+    ReferencedTask referencedTask = new ReferencedTask();
+    referencedTask.setsystemUrl(theValue);
+    assertThat(theValue).isEqualTo(referencedTask.getsystemUrl());
+  }
+
+  @Test
+  void should_returnTaskDefinitionKey_when_TaskDefinitionKeyWasSet() {
+    ReferencedTask referencedTask = new ReferencedTask();
+    referencedTask.setTaskDefinitionKey(theValue);
+    assertThat(theValue).isEqualTo(referencedTask.getTaskDefinitionKey());
+  }
+
+  @Test
+  void should_returnVariables_when_VariablesWasSet() {
+    ReferencedTask referencedTask = new ReferencedTask();
+    referencedTask.setVariables(theValue);
+    assertThat(theValue).isEqualTo(referencedTask.getVariables());
+  }
+
+  @Test
+  void should_returnDomain_when_DomainWasSet() {
+    ReferencedTask referencedTask = new ReferencedTask();
+    referencedTask.setDomain(theValue);
+    assertThat(theValue).isEqualTo(referencedTask.getDomain());
+  }
+
+  @Test
+  void should_returnClassificationKey_when_ClassificationKeyWasSet() {
+    ReferencedTask referencedTask = new ReferencedTask();
+    referencedTask.setClassificationKey(theValue);
+    assertThat(theValue).isEqualTo(referencedTask.getClassificationKey());
+  }
+
+  @Test
+  void should_returnWorkbasketKey_when_WorkbasketKeyWasSet() {
+    ReferencedTask referencedTask = new ReferencedTask();
+    referencedTask.setWorkbasketKey(theValue);
+    assertThat(theValue).isEqualTo(referencedTask.getWorkbasketKey());
+  }
+
+  @Test
+  void should_returnHashCode_when_askedToDoSo() {
+    assertThat(theTask.hashCode()).isNotNull();
+  }
+
+  @Test
+  void should_returnString_when_toStringIsCalled() {
+    assertThat(theTask.toString()).isNotNull();
+  }
+
+  @Test
+  void should_checkEgality_when_equalsIsCalled() {
+    ReferencedTask refTask2 = theTask;
+    refTask2.setWorkbasketKey("nochnkey");
+    assertThat(refTask2.equals(theTask));
+    refTask2 = theTask;
+    assertThat(theTask.equals(refTask2));
+    assertThat(!theTask.equals("aString"));
```

Comment:
please complete assertion, see above. Also maybe you could use the isNotEqualTo assertion like this: assertThat(theTask).isNotEqualTo("aString)